src/Application/FileProcessor/PhpFileProcessor.php

```diff
@@ -186,9 +203,6 @@ private function printFile(File $file, Configuration $configuration): void
         }
 
         $file->changeFileContent($newContent);
-        if ($configuration->shouldShowDiffs()) {
-            $this->fileDiffFileDecorator->decorate([$file]);
```

Comment:
generation of the file-diff (comparing contents, generate a diff) is slow.

therefore we just remember all other information while in flight and do the actual content comparison/diff generation only once after the loop